### ISSUE TITLE
Missing math symbols with OpenSans and lualatex (unicode)

### ISSUE THREAD

**Issue by schmittlauch** [tud-cd/tud-cd#28]
related to #23 

When using the tud beamer class in its cd2018 version, several math symbols like \beta or \Rightarrow are missing.
I use lualatex with fontspec as my engine, and have installed both the opensans and xits LaTeX packages as well as the system-wide opensans, xits and stix TTF/OTF fonts.

I get several font-related warnings from lualatex (excerpt):

```
 This is LuaTeX, Version 1.07.0 (Web2C 2018/NixOS.org) 
luaotfload | main : initialization completed in 0.093 seconds

(/nix/store/zdzyb20bdnap8s3x3g8761dlqwpvrl11-texlive-combined-2018/share/texmf/
tex/latex/beamer/beamer.cls
Document Class: beamer 2018/12/02 v3.55 A class for typesetting presentations


(/nix/store/zdzyb20bdnap8s3x3g8761dlqwpvrl11-texlive-combined-2018/share/texmf/
tex/latex/amsfonts/umsb.fd)

Package fontspec Warning: Font "OpenSans" does not contain requested Script
(fontspec)                "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSans' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSans' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: Font "OpenSansItalic" does not contain requested
(fontspec)                Script "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSansItalic' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSansItalic' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: Font "OpenSansBold" does not contain requested
(fontspec)                Script "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSansBold' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSansBold' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: Font "OpenSansBoldItalic" does not contain requested
(fontspec)                Script "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSansBoldItalic' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSansBoldItalic' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: Font "OpenSansCondensedLight" does not contain
(fontspec)                requested Script "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSansCondensedLight' with
(fontspec)                script 'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSansCondensedLight' with
(fontspec)                script 'CustomDefault' and language 'Default'.



Package fontspec Warning: Font "OpenSansCondensedLightItalic" does not contain
(fontspec)                requested Script "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSansCondensedLightItalic'
(fontspec)                with script 'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSansCondensedLightItalic'
(fontspec)                with script 'CustomDefault' and language 'Default'.



Package fontspec Warning: Font "OpenSansLight" does not contain requested
(fontspec)                Script "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSansLight' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSansLight' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: Font "OpenSansLightItalic" does not contain
(fontspec)                requested Script "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSansLightItalic' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSansLightItalic' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: Font "OpenSans" does not contain requested Script
(fontspec)                "Math".



Package unicode-math Warning: The first font loaded by unicode-math must be an
(unicode-math)                OpenType Math font (with script=math). If you
(unicode-math)                simply want âthe defaultâ before loadin
g
(unicode-math)                supplementary fonts over the top for certain
(unicode-math)                ranges, use: 
(unicode-math)                  \setmathfont{latinmodern-math.otf}



Package fontspec Warning: Font "OpenSans" does not contain requested Script
(fontspec)                "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSans' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSans' with script
(fontspec)                'CustomDefault' and language 'Default'.

) (/home/spiollinux/texmf/tex/latex/tud-beamerstyle/beamerouterthemetud.sty
(/nix/store/zdzyb20bdnap8s3x3g8761dlqwpvrl11-texlive-combined-2018/share/texmf/
tex/latex/tools/calc.sty))

 4147 words of node memory still in use:
   59 hlist, 22 vlist, 8 rule, 15 disc, 7 local_par, 114 glue, 10 kern, 27 pena
lty, 131 glyph, 135 attribute, 81 glue_spec, 135 attribute_list, 5 write, 30 pd
f_literal, 3 pdf_start_link, 3 pdf_end_link, 2 pdf_dest, 4 pdf_action, 28 pdf_c
olorstack nodes
   avail lists: 1:4,2:1146,3:719,4:427,5:127,6:112,7:3889,8:129,9:707,10:15,11:
248,12:1
</nix/store/sxhmyrfdzchmw26bmbclc9idgxlzc6zb-texlive-lm-2.004/fonts/opentype/pu
blic/lm/lmmono10-regular.otf></home/spiollinux/texmf/fonts/truetype/public/Open
Sans-Italic.ttf></nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/sha
re/fonts/truetype/OpenSans-Regular.ttf></nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6
hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-Italic.ttf></nix/store/ll2jmb
zbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-Bold.tt
f></nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truet
ype/OpenSans-Regular.ttf></nix/store/zdzyb20bdnap8s3x3g8761dlqwpvrl11-texlive-c
ombined-2018/share/texmf/fonts/type1/public/ccicons/ccicons.pfb>
Output written on talk-slides.pdf (23 pages, 1496331 bytes).
Transcript written on talk-slides.log.
```

My current workaround for getting at least all symbols displayed, although probably not in the correct font:

```
\usepackage{fontspec}
\setmainfont{TeX Gyre Pagella}
\RequirePackage{unicode-math}
  \setmathfont{XITS Math}
  \setmainfont{Open Sans}
  \setsansfont{Open Sans}
```

What am I doing wrong here?

**Comment by keinstein** [comment]
Please, provide a minimal working example. I just added greek letters to the example presentation and it worked out of the box. So I need a proper example that shows me what you are doing.

Do you have TUD-Script installed and accessible by your lualatex run?

Background: There are different font interfaces. In order to avoid duplicating activities one option is to use the font configuration of the TUD-Script bundle.

**Comment by schmittlauch** [comment]
relatively minimal example:

```tex
% $Header$
% use lualatex for compilation

\documentclass[aspectratio=169,navbar=false]{beamer}
\mode<presentation>
{
  \usetheme[cd2018,noddc,darktitlepage]{tud}
  \usecolortheme{tud}
  % or ...

  %\setbeamercovered{transparent}
  % or whatever (possibly just delete it)
}

% notes on 2nd screen:
\usepackage{pgfpages}
\setbeameroption{show notes on second screen}

\usepackage[british]{babel}
% or whatever

\usepackage[backend=biber, sorting=none]{biblatex}

%\usepackage{fontspec}
%\setmainfont{TeX Gyre Pagella}
%\RequirePackage{unicode-math}
%  \setmathfont{XITS Math}
%  \setmainfont{Open Sans}
%  \setsansfont{Open Sans}

\title[ex]
{Example}

\author
{anon}

\institute[] % (optional, but mostly needed)
{
  Department of Computer Science\\
  Technical University Dresden
}

\begin{document}

\maketitle

\begin{frame}{test}
	\(a \beta \Rightarrow\)
\end{frame}

\end{document}
```

> Do you have TUD-Script installed and accessible by your lualatex run?

Yes, although as a disclaimer my setup is a bit unusual: I'm using the Nix package manager to create a LaTeX environment, my current setup can be found [here](https://git.orlives.de/schmittlauch/paper_hashtag_federation/src/branch/master/shell.nix).
If you're interested in using Nix to reproduce an exact copy of that environment let me know, I'm glad to help (though not expecting people to deal with my weird architecture choices).

**Comment by keinstein** [comment]
I have no idea, can you provide me a complete log file?
Did you check out the latest revision of the tud-cd package? Is it correctly found by your TeX system or do you have an old version which is interfering?

**Comment by schmittlauch** [comment]
I'm using rev ff22134151d5e0afcfb3a0a583b4bff752d03a82 which is the latest one from master. There's definitely no old version interfering as Nix takes care of that.

[full log file](https://github.com/tud-cd/tud-cd/files/3556117/font.log)

**Comment by schmittlauch** [comment]
You were right, some remaining files in ~/texmf/ polluted my TeX environment. But removing them made things even worse:

```
Package fontspec Warning: Font "OpenSans" does not contain requested Script
(fontspec)                "Math".



Package fontspec Warning: OpenType feature 'Style=MathScript' (ssty) not
(fontspec)                available for font 'OpenSans' with script
(fontspec)                'CustomDefault' and language 'Default'.



Package fontspec Warning: OpenType feature 'Style=MathScriptScript' (ssty) not
(fontspec)                available for font 'OpenSans' with script
(fontspec)                'CustomDefault' and language 'Default'.


! Package fontspec Error: The font "OpenSansItalic" cannot be found.
```

Not even my workaround works anymore.
It's not unlikely that this issue is NixOS specific. But if you know more about font-discovery and fontconfig, please let me know. I'm planning to provide a working NixOS package for tud-cd.

`fc-list` doesn't know a font called "OpenSansItalic":
```
$ fc-list|grep -i opensans 
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-CondLightItalic.ttf: Open Sans,Open Sans Condensed Light:style=Condensed Light Italic,Italic
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-CondBold.ttf: Open Sans Condensed:style=Bold
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-LightItalic.ttf: Open Sans,Open Sans Light:style=Light Italic,Italic
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-ExtraBold.ttf: Open Sans,Open Sans Extrabold:style=Extrabold,Regular
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-Light.ttf: Open Sans,Open Sans Light:style=Light,Regular
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-CondLight.ttf: Open Sans,Open Sans Condensed Light:style=Condensed Light,Regular
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-Bold.ttf: Open Sans:style=Bold
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-BoldItalic.ttf: Open Sans:style=Bold Italic
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-Italic.ttf: Open Sans:style=Italic
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-Semibold.ttf: Open Sans,Open Sans Semibold:style=Semibold,Regular
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-Italic.ttf: Open Sans:style=Italic
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-SemiboldItalic.ttf: Open Sans,Open Sans Semibold:style=Semibold Italic,Italic
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-Light.ttf: Open Sans,Open Sans Light:style=Light,Regular
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-Regular.ttf: Open Sans:style=Regular
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-CondBold.ttf: Open Sans Condensed:style=Bold
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-ExtraBoldItalic.ttf: Open Sans,Open Sans Extrabold:style=Extrabold Italic,Italic
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-Semibold.ttf: Open Sans,Open Sans Semibold:style=Semibold,Regular
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-Regular.ttf: Open Sans:style=Regular
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-Bold.ttf: Open Sans:style=Bold
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-BoldItalic.ttf: Open Sans:style=Bold Italic
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-CondLightItalic.ttf: Open Sans,Open Sans Condensed Light:style=Condensed Light Italic,Italic
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-LightItalic.ttf: Open Sans,Open Sans Light:style=Light Italic,Italic
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-ExtraBold.ttf: Open Sans,Open Sans Extrabold:style=Extrabold,Regular
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-SemiboldItalic.ttf: Open Sans,Open Sans Semibold:style=Semibold Italic,Italic
/home/spiollinux/.nix-profile/share/fonts/truetype/OpenSans-CondLight.ttf: Open Sans,Open Sans Condensed Light:style=Condensed Light,Regular
/nix/store/ll2jmbzbisvgs2sl02m2wp2w6kj6hpxa-open-sans-1.11/share/fonts/truetype/OpenSans-ExtraBoldItalic.ttf: Open Sans,Open Sans Extrabold:style=Extrabold Italic,Italic
```

The OpenSans package I am using takes the same font file as Debian does, so that shouldn't be an issue.

So is the required font really called "Open Sans Italic" or "OpenSansItalic"?

**Comment by schmittlauch** [comment]
current workaround for `beamerfontthemetud.sty`:

```diff157,164c157,164
<       \setmathfont[range={it}]{Open Sans Italic}
<       \setmathfont[range={bfup}]{Open Sans Bold}
<       \setmathfont[range={bfit}]{Open Sans Bold Italic}
<       \setmathfont[range={sfup}]{Open Sans Condensed Light}
<       \setmathfont[range={sfit}]{Open Sans Condensed Light Italic}
<       \setmathfont[range={bfsfup}]{Open Sans Light}
<       \setmathfont[range={bfsfit}]{Open Sans Light Italic}
<       \setmathfont[range=\mathfrak]{STIXGeneral}
---
>       \setmathfont[range={it}]{Open Sans}
>       \setmathfont[range={bfup}]{Open Sans}
>       \setmathfont[range={bfit}]{Open Sans}
>       \setmathfont[range={sfup}]{Open Sans}
>       \setmathfont[range={sfit}]{Open Sans}
>       \setmathfont[range={bfsfup}]{Open Sans}
>       \setmathfont[range={bfsfit}]{Open Sans}
>         %\setmathfont[range=\mathfrak]{STIX Math}
```

**Comment by mrpiggi** [comment]
Actually, `beamerfontthemetud.sty` should be updated to use package `tudscrfonts`.This would be much easier than trying to do all the font specification by hand. A fast workaround would be to switch from `cd2018` to `cd2012ts` as this would load package `tudscrfonts` which is set up for using Open Sans by default. But I don't know, what other effects could occur. If you are using LuaLaTeX, loading package `fontspec` is necessary.
```tex
% $Header$
% use lualatex for compilation
\documentclass[aspectratio=169,navbar=false,british]{beamer}
\mode<presentation>
{
  \usetheme[cd2012ts,noddc,darktitlepage]{tud}
  \usecolortheme{tud}
}
\usepackage{fontspec}
\usepackage{babel}
\title[ex]
{Example}

\author
{anon}

\institute[] % (optional, but mostly needed)
{
  Department of Computer Science\\
  Technical University Dresden
}

\begin{document}
\maketitle
\begin{frame}{test}
	\(a \beta \Rightarrow\)
\end{frame}
\end{document}
```
[Result](https://github.com/tud-cd/tud-cd/files/3557721/tudbeamer.pdf)

**Comment by keinstein** [comment]
`fontspec` is loaded by `beamerouterthemetud.sty`
So this cannot be the error.

The current Corporate Design (cd2018) looks completely different than the old one. So switching the standard is not an option.

You can try the option `beamerfont` and loading `tudscrfonts.sty` with the option `heavyfont` but this does **not** fit the requirements of the corporate design for presentations.

```
% $Header$
% use lualatex for compilation
\documentclass[british]{beamer}
\mode<presentation>
{
  \usetheme[cd2018,beamerfont]{tud}
  \usepackage{fontspec}
  \usepackage[cdfont=heavyfont]{tudscrfonts}
%  \usecolortheme{tud}% only needed if other themes are loaded inbetween like
%                                       \useinnertheme{rounded}
}
\usepackage{babel}
\title[ex]
{Example}

\author
{anon}

\institute[] % (optional, but mostly needed)
{
  Department of Computer Science\\
  Technical University Dresden
}

\begin{document}
\maketitle

\begin{frame}{test}
	\(a \beta \Rightarrow\)
\end{frame}
\end{document}
```

Back to the orignial problem: It seems as if the latex distribution does not contain an italic otf font. Even `tudscrfonts.sty` falls back to a Type1 font when fontspec uses OpenType (This may cause problems for languages that depend on OpenType features like Serbian cyrillic). At least debian provides OpenType fonts that contain the missing shapes in the package `fonts-open-sans`.  

I still don't understand why LuaLaTeX seems to be ignoring the line `\setmathfont{XITS Math}` while the rest seems to be executed. And why it works when schmittlauch copies the lines into the main body of the tex file.

**Comment by keinstein** [comment]
I played around with tudscrfonts, the option `cdfont=ultrabold` seems to be helpful in this setting. This lead to another issue with `tudscrfonts.sty`. See  https://github.com/tud-cd/tudscr/issues/43 for a minimal example for both the setting and the issue.

**Comment by keinstein** [comment]
Another workaround: The TUD shareLaTeX distribution contains the presentation style. https://tex.zih.tu-dresden.de/

**Comment by mrpiggi** [comment]
> `fontspec` is loaded by `beamerouterthemetud.sty`
> So this cannot be the error.

Without loading it manually, I get an error `\textfont7 is undefined (character 97)`

> The current Corporate Design (cd2018) looks completely different than the old one. So switching the standard is not an option.

As I said, an update of `beamerfontthemetud.sty` would be the best solution. Setting default font options like `cdfont=ultrabold` should be done with `\TUDoptions` after loading.

> Back to the orignial problem: It seems as if the latex distribution does not contain an italic otf font. Even `tudscrfonts.sty` falls back to a Type1 font when fontspec uses OpenType (This may cause problems for languages that depend on OpenType features like Serbian cyrillic). At least debian provides OpenType fonts that contain the missing shapes in the package `fonts-open-sans`.

I'm not sure, if I get you right. `tudscrfonts.sty` uses Type1 font for math mode as Open Sans does not support OpenType features for math. And package `opensans` provides italic shapes both for Type1 and OpenType fonts. 

> **Update:** The opensans font package provides all files in otf fonts.

Obviously you found these. ;)

**Comment by mrpiggi** [comment]
> Another workaround: The TUD shareLaTeX distribution contains the presentation style. https://tex.zih.tu-dresden.de/

This is actually not an option as ShareLaTeX/Overleaf uses Tex Live 2017. So there are no updated packages since more than a year, which also concerns `tudscrfonts.sty`

**Comment by keinstein** [comment]
> > Another workaround: The TUD shareLaTeX distribution contains the presentation style. https://tex.zih.tu-dresden.de/
> 
> This is actually not an option as ShareLaTeX/Overleaf uses Tex Live 2017. So there are no updated packages since more than a year, which also concerns `tudscrfonts.sty`

Just to stay on topic: This is a topic about some strange font problem that is related to an unusual TeX setup and to tud-beamerstyle, but not about any TUD Script package. Other topics should be discussed in separate topics.

As you realized `tudscrfonts.sty` is not used by the current beamer style. So it is simply not relevant for a workaround whether it is up to date or not. But the beamer presentation example seems to work even with greek letters.

@schmittlauch Do you have the opensans LaTeX package installed? It is not directly mentioned in your setup. If not, you should get an error message. But that may be one reason why the XITS font is not loaded.

**Comment by schmittlauch** [comment]
> Do you have the opensans LaTeX package installed? It is not directly mentioned in your setup. If not, you should get an error message.

I have both the LaTeX package and the system package installed, but the system package only provides the .ttf fonts.
If Type1 fonts are absolutely required, that might be the reason for this issue. I'll have a look into the used search paths.

**Comment by mrpiggi** [comment]
> As you realized `tudscrfonts.sty` is not used by the current beamer style. So it is simply not relevant for a workaround whether it is up to date or not. 

Well, it does use the package `tudscrfonts.sty` with option `cd2012ts` and then uses not Univers but Open Sans. At least, option `cdoldfont` should be passed in this case. In general, I just mentioned it as font setup is a very fragile issue and this packages tries to deal with most of it. But never mind.

**Comment by keinstein** [comment]
No, in the current setting (LuaLaTeX with OpenSans) `tudscrfonts.sty` is not loaded from `beamerthemetud.sty` or any of the packages used by that. You can verify this by running the example from https://github.com/tud-cd/tud-cd/issues/28#issuecomment-526194527 through `lualatex` and checking the logs.

This thread is only about schmittlauchs problem.

Discussing the interplay between `tudscrfonts.sty` and `beamerfontthemetud.sty` deserves its own issue. BTW. it needs https://github.com/tud-cd/tudscr/issues/43 to be solved at first. So, please move this discussion to another issue and let's solve the orignal one, here.

**Comment by keinstein** [comment]
> > Do you have the opensans LaTeX package installed? It is not directly mentioned in your setup. If not, you should get an error message.
> 
> I have both the LaTeX package and the system package installed, but the system package only provides the .ttf fonts.
> If Type1 fonts are absolutely required, that might be the reason for this issue. I'll have a look into the used search paths.

That's strange. Fontconfig should find the system .ttf files, which should include the italic shape. Consequently LuaTeX should find the italic shape regardless of the version in your home directory.
Did you rescan your fontconfig directories? Or is this a byproduct of the application separation in the nix package manager.

**Comment by mrpiggi** [comment]
As I said, with option **`cd2012ts`**, package `tudscrfonts` is used! Search the log file for it. I just made the suggestion to pass option `cdoldfont` to `tudscrfonts` _in this case_. Or just remove option `cd2012ts` if you don't want to support the old corporate design anymore.
```tex
% $Header$
% use lualatex for compilation
\listfiles
%\PassOptionsToPackage{cdoldfont}{tudscrfonts}
\documentclass{beamer}
\mode<presentation>
{
  \usetheme[cd2012ts]{tud}
%  \usecolortheme{tud}
}
\usepackage{fontspec}
\begin{document}
\begin{frame}{test}
	\(a \beta \Rightarrow\)
\end{frame}
\end{document}
```

**Comment by keinstein** [comment]
As I said `cd2012ts` is not an option to be discussed here! It is not the current CD!

**Comment by schmittlauch** [comment]
I fixed the main issue in #33, but this issue thread here brought up some other relevant issues.

As of April 2018 the *stix* CTAN package is considered as deprecated, so if we want to keep on using STIX fonts switching to stix2 needs to be considered.

Also, updating beamerfontthemetud.sty  to use package tudscrfonts as suggested  here https://github.com/tud-cd/tud-cd/issues/28#issuecomment-526403880 might be an option.
